windows/ReactNativePicker/ReactPickerView.cpp

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "JSValueXaml.h"
+#include "ReactPickerView.h"
+#include "ReactPickerView.g.cpp"
+
+namespace winrt {
+    using namespace Microsoft::ReactNative;
+    using namespace Windows::Foundation;
+    using namespace Windows::Foundation::Metadata;
+    using namespace Windows::UI;
+    using namespace Windows::UI::Xaml;
+    using namespace Windows::UI::Xaml::Controls;
+    using namespace Windows::UI::Xaml::Input;
+    using namespace Windows::UI::Xaml::Media;
+} // namespace winrt
+
+namespace winrt::ReactNativePicker::implementation {
+
+    ReactPickerView::ReactPickerView(winrt::IReactContext const& reactContext) : m_reactContext(reactContext) {
+        m_isEditableComboboxSupported = winrt::ApiInformation::IsPropertyPresent(
+            L"Windows.UI.Xaml.Controls.ComboBox", L"IsEditableProperty");
+        this->AllowFocusOnInteraction(true);

```

Comment:
do we have other usages of WUX types? we should collect these somewhere to make it easier to migrate to WinUI3
e.g.
#define XAML_NAMESPACE  L"Windows.UI.Xaml"
#define MAKE_XAML_TYPE(x) XAML_NAMESPACE ## L"." ## L#x

then you can pass MAKE_XAML_TYPE(Controls.ComboBox)

when WinUI3 comes along we just change XAML_NAMESPACE to L"Microsoft.UI.Xaml"